betterttv.js

```diff
@@ -4098,7 +4078,449 @@ module.exports = function () {
+
+module.exports = function () {
+    if(bttv.settings.get("formatTeamPage") !== true || $("body").attr("data-page") != "teams#show") return;
+    
+    vars.teamCurrentChannel = ($(".js-playing").attr("id")).replace("channel_", "");
+    vars.teamFirstLoad = 1;
+    vars.teamName = window.location.pathname.replace(/\/team\/|\/event\//, "");
+    
+    var teamFollowList = "none",
+        teamFollowedFailedList = "none",
+        teamFollowRequestCount = 0,
+        teamFollowResponseCount = 0,
+        teamFollowInterval = "none";
+        
+    $(window).resize(function() {
+        var vpWidth = $(window).width(),
+            vpHeight = $(window).height(),
+            paddedWidth = vpWidth - 40,
+            paddedHeight = vpHeight - ($("#site_header").outerHeight(true) + 40),
+            teamListHeight = paddedHeight - ($("#team_logo").outerHeight(true) + $("#bttvFollowTeamButton").outerHeight(true) + 2),
+            playerWidth = paddedWidth - ($(".filters.grid.c4").outerWidth(true) + $("#bttvTeamChat").outerWidth(true) + 2),
+            arMultiplier = (playerWidth) / 16,
+            playerHeight = (9 * arMultiplier) + 32;
+            
+        if(paddedWidth < 985) {
+            paddedWidth = 985;
+        }
+
+        $("div.main.discovery.team").css("width", vpWidth+"px");
+        $("div.wrapper.c12.clearfix").css("width", paddedWidth+"px");
+        $("#player_column").css({"width":(playerWidth+2)+"px", "height":paddedHeight+"px"});
+        $("#site_footer").css("width", playerWidth+"px");
+        $("#standard_holder").css({"width":playerWidth+"px", "height":playerHeight+"px"});
+        $("#bttvTeamChat").css("height", (paddedHeight-2)+"px");
+        $("#bttvTeamMemberListOuter").css({"height":teamListHeight+"px"});
+    });
+    
+    var betterTeamInit = function() {
+        var teamCSS = $("<link>", {"href":"//cdn.betterttv.net/style/stylesheets/betterttv-team-page.css?"+bttv.info.versionString(), "id":"betterTwitchTeams", "rel":"stylesheet", "type":"text/css"});
+        
+        if(bttv.settings.get("darkenedMode") === true) {
+            teamCSS = $("<link>", {"href":"//cdn.betterttv.net/style/stylesheets/betterttv-team-page-dark.css?"+bttv.info.versionString(), "id":"betterTwitchTeams", "rel":"stylesheet", "type":"text/css"});
+        }
+        $("body").append(teamCSS);
+        
+        getFeatured();
+        formatTeamPage();
+        //if it wasn't for the bttv.TwitchAPI._ref is null error in Firefox, I would just load the team without the timeout
+        setTimeout(loadTeam, 1000);
+    }
+    
+    var getFeatured = function() {
+        bttv.TwitchAPI.get("/kraken/streams/featured?limit=1")
+        .done(function(d) {
+            //debug.log("got preview url template on load");
+            var chanName = d.featured[0].stream.channel.name;
+            var updatedPreviewUrl = d.featured[0].stream.preview;
+            vars.updatedPreviewUrl = updatedPreviewUrl.replace(chanName, "CHANNEL");
```

Comment:
If that's the functionality, I suggest changing to instead pull from ``kraken/streams?channel=channel1,channel2,channel3``. This might make more sense than replacing a channel name in the url?

There's also an odd edge case to doing what your doing right now. What if the user's username is in the URL twice?